code/datums/keybinding/living.dm

```diff
@@ -89,3 +89,31 @@
 		return
 	var/mob/living/user_mob = user.mob
 	user_mob.set_combat_mode(!user_mob.combat_mode, FALSE)
+
+/datum/keybinding/living/enable_combat_mode
+	hotkey_keys = list("Unbound")
+	name = "enable_combat_mode"
+	full_name = "Enable Combat Mode"
+	description = "Enable combat mode. Like Help/Harm but lamer."
```

Comment:
Come on.

Reply:
In my defense, I meant to change these before committing, but forgot.

Reply:
Just realizing Floyd wrote this himself. He shouldn't have 🙂